### PR TITLE
Lifting of embargo now stores item in embargo_history

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
- When embargo is lifted through either the UI or background job, an entry is addded to embargo_history
Fixes: #1219
Example text: "embargo_history_ssim":["An embargo was deactivated on 2019-08-22 12:07:53 -0600. Its release date was 2023-01-01T00:00:00.000Z. Intended visibility after embargo was http://terms.library.ualberta.ca/public"]

### CODE REVIEW

File: test/system/deposit_item_test.rb [hunk 1]
```diff
@@ -84,6 +84,11 @@ def before_all
     assert_text I18n.t('items.draft.successful_deposit')
     assert_selector 'h1', text: Item.last.title
 
+    # Check to make sure there isn't any embargo_history
+    item_id = current_url.split('/')[-1]
```
Comment (pgwillia) [inline test/system/deposit_item_test.rb]: I wasn't sure what the [-1] was doing here until I looked it up.  Could do 
```suggestion
    item_id = current_url.split('/').last
```

Comment (pgwillia) [reply test/system/deposit_item_test.rb]: @ConnorSheremeta do you have any thoughts about this?

Comment (ConnorSheremeta) [reply test/system/deposit_item_test.rb]: Using .last is definitely more clear in what it's doing, committing your suggestion

File: test/system/deposit_thesis_test.rb [hunk 2]
```diff
@@ -82,6 +82,11 @@ def before_all
     assert_text I18n.t('admin.theses.draft.successful_deposit')
     assert_selector 'h1', text: Thesis.last.title
 
+    # Check to make sure there isn't any embargo_history
+    item_id = current_url.split('/')[-1]
```
Comment (pgwillia) [inline test/system/deposit_thesis_test.rb]: One more occurance
```suggestion
    item_id = current_url.split('/').last
```

**Pull request merged**